mssql-mixin/config.libsonnet

```
@@ -1,21 +1,39 @@
 {
-  _config+:: {
-    enableMultiCluster: false,
-    multiclusterSelector: 'job=~"$job"',
-    mssqlSelector: if self.enableMultiCluster then 'job=~"$job", cluster=~"$cluster"' else 'job=~"$job"',
-    dashboardTags: ['mssql-mixin'],
-    dashboardPeriod: 'now-1h',
-    dashboardTimezone: 'default',
-    dashboardRefresh: '1m',
+  local this = self,
+  enableMultiCluster: false,
+  filteringSelector: 'job=~"integrations/mssql"',
+  groupLabels: ['job', 'cluster'],  // Remove 'db' from here
+  instanceLabels: ['instance'],
 
-    // alert thresholds
-    alertsWarningDeadlocks5m: 10,
-    alertsWarningModerateReadStallTimeMS: 200,
-    alertsCriticalHighReadStallTimeMS: 400,
-    alertsWarningModerateWriteStallTimeMS: 200,
-    alertsCriticalHighWriteStallTimeMS: 400,
 
-    // enable Loki logs
-    enableLokiLogs: true,
+  dashboardTags: [self.uid],
+  legendLabels: ['instance'],
+  uid: 'mssql',
+  dashboardNamePrefix: 'MSSQL',
+
+  // additional params
+  dashboardPeriod: 'now-1h',
+  dashboardTimezone: 'default',
+  dashboardRefresh: '1m',
+  metricsSource: 'prometheus',
+
+  // logs lib related
+  enableLokiLogs: true,
+  logLabels: ['job', 'cluster', 'instance'],
```

```suggestion
  logLabels: self.groupLabels + self.instanceLabels,
```
would it be a better default?